coding-standard/action.yml

```diff
@@ -79,15 +89,21 @@ runs:
       run: composer config allow-plugins.dealerdirect/phpcodesniffer-composer-installer  true --global
       if: steps.is-allow-plugins-available.outputs.result < 1
 
-    - name: Install Coding Standard
+    - name: Install composer require dealerdirect/phpcodesniffer-composer-installer
+      shell: bash
+      working-directory: standard
+      run: composer require dealerdirect/phpcodesniffer-composer-installer
+
+    - name: Install Magento2 Coding Standard
       shell: bash
       working-directory: standard
       run: composer require "magento/magento-coding-standard:${{ inputs.version || '*' }}"
 
-    - name: Register Coding Standard
+    - name: Install Custom Coding Standard
```

Comment:
I foresee two situations for this action to be used:

1. Extensions
2. Stores

both of them will often have this extra repo already included in `require-dev`. Should we attempt to re-install this it seems like an unnecessary network IO? Perhaps #133 applies to this as well?

I don't personally `custom_coding_standard_repo` is a good idea.